src/features/SendAssets/components/fields/GasPriceSlider.tsx

```diff
@@ -11,21 +11,13 @@ import './GasPriceSlider.scss';
 
 const SliderWithTooltip = createSliderWithTooltip(Slider);
 
-interface OwnProps {
+interface Props {
   gasPrice: string;
   gasEstimates: GasEstimates;
   network: Network;
   onChange(value: number): void;
 }
 
-type Props = OwnProps;
-
-interface State {
-  gasPrice: string;
-  hasSetRecommendedGasPrice: boolean;
```

Comment:
jolean correction not boolean

Reply:
Hi! Could you stop spamming and leaving random reviews on our PRs? If you need help with anything please email support@mycrypto.com